src/services/doppler-legacy-client.ts

```diff
@@ -125,21 +125,14 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
   }
 
   public async getUserData() {
-    var response = await this.axios.get('/Reports/Reports/GetUserData');
-    if (!response || !response.data || response.data.Email) {
+    var response = await this.axios.get('/WebApp/GetUserData');
+    if (!response || !response.data) {
       throw new Error('Empty Doppler response');
     }
+    if (response.data.error) {
+      throw new Error(`Doppler Error: ${response.data.error}`);
+    }
 
-    // TODO: get this data from backend
-    const { alert, nav, user } = mapHeaderDataJson(headerDataJson);
```

Comment:
Did you remove the dependency on `headerDataJson`?

Reply:
No, removing now...